egs/ifnenit/v1/cmd.sh

```diff
@@ -0,0 +1,27 @@
+# "queue.pl" uses qsub.  The options to it are
```

Comment:
Please make this file like iam/v1/cmd.sh and update `$train_cmd` and `$decode_cmd` to `$cmd` in your scripts.